### ISSUE TITLE
[Feature]Rename seatunnel-engine module

### ISSUE THREAD

**Issue by hailin0** [apache/seatunnel#3857]
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Currently we have rename `seatunnel-engine` to `seatunnel-zeta` (voting in the email), and we need to change the name in the code repository:
- [ ] update maven module names
    - seatunnel-engine/*
    - seatunnel-examples/seatunnel-engine-examples/*
    - seatunnel-e2e/seatunnel-engine-e2e/*
    - seatunnel-core/seatunnel-starter/*
- [ ] update class package names
    - seatunnel-engine/*
    - seatunnel-examples/seatunnel-engine-examples/*
    - seatunnel-e2e/seatunnel-engine-e2e/*
    - seatunnel-core/seatunnel-starter/*
- [ ] update docs

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by Sachin-2206** [comment]
Sir, I want to contribute in this. How can I contribute in this. tell me about this more,

**Comment by TaoZex** [comment]
> Sir, I want to contribute in this. How can I contribute in this. tell me about this more,

https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/contribution
Here are some contributed documents that may be helpful to you. If you have any questions, please feel free to contact us.

**Comment by hailin0** [comment]
> Sir, I want to contribute in this. How can I contribute in this. tell me about this more,


@Sachin-2206 Need to be assigned to you?

**Comment by Sachin-2206** [comment]
Yes sir... I want to contribute on this. I will give my full potential.
Thank you
.

**Comment by Sachin-2206** [comment]
Thanks a lot.

**Comment by Sachin-2206** [comment]
Sir I updated maven module names. Sir should I also update the class package names ?( seatunnel engine to seatunnel zeta )

**Comment by Sachin-2206** [comment]
Sir I want to know that should I change only package name or complete import statements names ?

**Comment by Sachin-2206** [comment]
Sir I completed my work . I want to send a pull request of it . Please  check it and tell me whether my work is correct or not,

**Comment by hailin0** [comment]
> Sir I updated maven module names. Sir should I also update the class package names ?( seatunnel engine to seatunnel zeta )

updates:
- maven moduel name & dir name
- class package name

**Comment by hailin0** [comment]
<img width="668" alt="image" src="https://user-images.githubusercontent.com/14371345/211831493-3211a3da-8d25-4e86-84de-87941713bd90.png">
don't update these files @Sachin-2206

**Comment by kanha-gupta** [comment]
@hailin0 can i work on this if there's no action?

**Comment by hailin0** [comment]
Anyone willing to fix it?
